egs/mgb2/asr/local/score_sclite.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
```

Comment:
We do not have to include this.
Please remove it.

Reply:
done